eng/pipelines/partner-release.yml

```diff
@@ -11,73 +11,80 @@ parameters:
   type: boolean
   default: true
 
-resources:
-  repositories:
-    - repository: azure-sdk-build-tools
-      type: git
-      name: internal/azure-sdk-build-tools
-      ref: refs/tags/azure-sdk-build-tools_20230829.1
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage:
+        displayName: 'Partner Release'
+        variables:
+          - name: BuildToolScripts
+            value: $(Pipeline.Workspace)/azure-sdk-build-tools/scripts
+          - name: Artifacts
+            value: $(Pipeline.Workspace)/artifacts
+          - name: EsrpArtifacts
+            value: $(Build.SourcesDirectory)/packages-esrp
```

Comment:
This changed from `$(Pipeline.Workspace)/packages-esrp` -> `$(Build.SourcesDirectory)/packages-esrp`.

I tried everything, including copying the downloaded unsigned artifacts INTO the `packages-esrp` as a subfolder. Still doesn't scan it properly.

The instant I moved it under `$(Build.SourcesDirectory)`, binskim errors disappeared.